benchmark_runner/grafana/perf/Makefile

```diff
@@ -0,0 +1,5 @@
+apply:
+	@cd jsonnet/ && \
+		jb install && \
+		cd ../ && \
+		terraform apply -auto-approve | python3 extract_dashboard.py dashboard.json
```

Comment:
It would be best for extract_dashboard.py to simply send its output to stdout (so you could easily e. g. pipe it through `jq` if desired), but I'll accept this, as it avoids the temporary file.

Reply:
Yes, I approved it as such.  I still think that extract_dashboard should write to stdout rather than to a separate file; that would allow you to e. g. process it further through jq, but I've accepted it as it avoids the temporary file.

Reply:
Do you mean to dashboard.json file?
We must have this file because we use it to update Grafana dashboard in case of changed.

Reply:
Correct, but you could write it as

```
terraform apply -auto-approve | python3 extract_dashboard.py > dashboard.json
```

Then if you want at some point to process the JSON with jq downstream of extract_dashboard.py you can simply add it to the pipeline.

Reply:
You right, updated it !!!